lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -389,9 +420,10 @@ def main(self):
 
         if self.options.output:
             exitcode, _ = RunFileLoopMPI.run(self.mpirun_cmd, output_timeout=self.options.output_check_timeout,
-                                             filename=self.options.output)
+                                             filename=self.options.output, fatal_option=self.options.output_check_fatal)
         else:
-            exitcode, _ = RunAsyncMPI.run(self.mpirun_cmd, output_timeout=self.options.output_check_timeout)
+            exitcode, _ = RunAsyncMPI.run(self.mpirun_cmd, output_timeout=self.options.output_check_timeout,
+                                          fatal_option=self.options.output_check_fatal)
```

Comment:
there's lot of common stuff here, so how about something like:

```python
run_kwargs = {
    'fatal_option': self.options.output_check_fatal,
    'output_timeout': self.options.output_check_timeout,
}
if self.options.output:
    run_mpirun_cmd = RunFileLoopMPI.run
    run_kwargs.update({'filename': self.options.output})
else:
    run_mpirun_cmd = RunAsyncMPI.run
exitcode, _ = run_mpirun_cmd(self.mpirun_cmd, **run_kwargs)
```

but do come up with a better name than `fatal_option`... ;-)